notebooks/concepts/imviz_compass_mpl.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a8d6eac-6626-415d-ade4-3f986cf18a69",
+   "metadata": {},
+   "source": [
+    "# Imviz compass visualization using Matplotlib\n",
+    "\n",
+    "This concept notebook visualizes the compass algorithm that Imviz will use. For simplicity, visualization is done using Matplotlib.\n",
+    "\n",
+    "These examples are also in the unit tests for `wcs_utils.py`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e571e723",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gwcs\n",
+    "import numpy as np\n",
+    "from astropy import units as u\n",
+    "from astropy.coordinates import ICRS\n",
+    "from astropy.modeling import models\n",
+    "from astropy.wcs import WCS\n",
+    "from gwcs import coordinate_frames as cf\n",
+    "\n",
+    "from jdaviz.configs.imviz import wcs_utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd58f8ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3779ae77-97fa-49db-85e5-7e8213c0cd30",
+   "metadata": {},
+   "source": [
+    "## Case 1: Dummy WCS with 10x10 image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae33669d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = np.arange(100).reshape((10, 10))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c334e59e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = WCS()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20605e33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs_utils.draw_compass_mpl(a, w)"
```

Comment:
| This PR | Ginga |
| --- | --- |
| ![dummy_wcs_imviz](https://user-images.githubusercontent.com/2090236/144470393-4474bad9-9c16-4d2d-a1c8-5ca6cd1b3367.jpg) | ![dummy_wcs_ginga](https://user-images.githubusercontent.com/2090236/144470417-c06d92e1-9d73-4957-b422-b03ff4e4166a.jpg) |